traitsui/qt4/tests/test_ui_panel.py

```diff
@@ -110,3 +161,53 @@ def test_subpanel_no_toolbar_nor_button_in_widget(self):
 
             # No button
             self.assertIsNone(ui.control.findChild(QtGui.QPushButton))
+
+
+@requires_toolkit([ToolkitName.qt])
+class TestPanelLayout(unittest.TestCase):
+
+    def test_scrollable_group_typical(self):
+        from pyface.qt import QtGui
+
+        example = ScrollableGroupExample()
+
+        ui = example.edit_traits(view=scrollable_group_view)
+        mainwindow = ui.control.layout().itemAt(0).widget()
```

Comment:
Minor: If instead we do `mainwindow, = ui.control.findChildren(QtGui.QMainWindow)`, this may expose the test to less implementation details while still asserting the expected behaviour?

Reply:
While it's a bit verbose, I'm OK with this - it's saying find the first (and it turns out, only) item in the control's layout, and get its widget, which is a reasonably straightforward description of what we're looking for.

The most likely change to affect code would be to make the top-level control a `QMainWindow` subclass instead of a sticky `Dialog` in which case this test would be broken anyway.